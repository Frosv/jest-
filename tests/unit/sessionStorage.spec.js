/*
 * @Author: your name
 * @Date: 2021-01-11 14:13:36
 * @LastEditTime: 2021-02-03 20:55:51
 * @LastEditors: Please set LastEditors
 * @Description: In User Settings Edit
 * @FilePath: /vuetest/tests/unit/primary.spec.js
 */

import { shallowMount } from '@vue/test-utils'
import losessionStoragecation from '@/views/sessionStorage.vue'

describe('location', () => {
  const wrapper = shallowMount(losessionStoragecation)
  sessionStorage.setItem('user', 123)
  // await Vue.nextTick()
  it('测试快照', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('init', () => {
    wrapper.vm.init()
  })
})
