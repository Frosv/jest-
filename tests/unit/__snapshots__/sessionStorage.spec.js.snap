// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`location 测试快照 1`] = `<div>sessionStorage</div>`;
